CHANGELOG.md

```diff
@@ -1,3 +1,48 @@
+## [12.26.8](https://github.com/botpress/botpress/compare/v12.26.3...v12.26.8) (2021-12-02)
+
+
+### Bug Fixes
+
+* **1002:** make embed chat icon less weird in iframe ([#5640](https://github.com/botpress/botpress/issues/5640)) ([99b1a9b](https://github.com/botpress/botpress/commit/99b1a9b))
```

Comment:
Euhm that one is weird

Reply:
Yep ... it's actually how the pr was named 😵‍💫  I made sure it's ok in the release notes